test/run-tests.sh

```diff
@@ -17,7 +17,8 @@ echo "INFO: the following test groups will be run:"
 TESTS=($(go run -tags experimental scripts/test-wrapper.go))
 echo ${TESTS[*]}
 
-for g in ${TESTS[*]}
-do
-    go test -tags experimental -timeout 60m -race -v -run "$g"
-done
+#for g in ${TESTS[*]}
+#do
+#    go test -tags experimental -timeout 60m -race -v -run "$g"
+#done
+go test -tags experimental -timeout 60m -race -v -run "TestElasticsearchGroup"
```

Comment:
It would appear this was done for local debugging and this needs to be put back :thinking:

Reply:
Oh jeeze, yup. Thanks!